### PR TITLE
added phaser template

### PR DESCRIPTION
**Pull request by Enjoy2Live**

### CODE REVIEW

File: templates/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ After cloning the monorepo, install dependencies, link packages, and build:
 
 ```bash
 $ npm i # install all dependencies
-$ npx lerna boostrap --hoist # link all packages together
+$ npx lerna bootstrap --hoist # link all packages together
```
Comment (kevo1ution) [inline templates/README.md:14]: lmao good catch

**Review by kevo1ution — APPROVED** [review]

**Comment by kevo1ution** [comment]
LGTM! Going to merge after CI passes

**Review by yquansah — COMMENTED** [review]
LGTM looks like we can be consistent with either single quote strings or double quote strings in the `js` part but maybe that should be a CI thing

**Comment by kevo1ution** [comment]
> LGTM looks like we can be consistent with either single quote strings or double quote strings in the `js` part but maybe that should be a CI thing

good point, once we set up a linter for all of our templates, this will be easier to enforce

**Pull request merged**